Example/package.json

```diff
@@ -10,17 +10,23 @@
     "test": "jest"
   },
   "dependencies": {
-    "react": "16.3.1",
-    "react-native": "0.55.4",
-    "react-native-animatable": "^1.3.0",
-    "react-native-indicators": "^0.13.0",
-    "react-native-spinner-button": "^1.0.1"
+    "react": "16.13.1",
+    "react-native": "0.63.4",
+    "react-native-animatable": "^1.3.3",
+    "react-native-indicators": "^0.17.0",
+    "react-native-spinner-button": "^1.0.4"
```

Comment:
react-native-spinner-button in latest version 1.0.3 instedof 1.0.4

Reply:
changes are done.